js/exporter/jspdf/pdf_grid.js

```diff
@@ -130,8 +130,18 @@ export class PdfGrid {
             if(this._pages.indexOf(page) > 0) {
                 doc.addPage();
             }
+
+            page._tables.forEach((table) => {
+                table.drawTo(doc, styles, {
+                    skipDrawingCustomBorders: true
+                });
+            });
+
             page._tables.forEach((table) => {
-                table.drawTo(doc, styles);
+                table.drawTo(doc, styles, {
+                    skipDrawingBorders: true,
+                    skipDrawingCellContent: true
+                });
```

Comment:
What do you think?

            page._tables.forEach((table) => {
                table.drawTo(doc, styles, {
                    allowDrawBorders: true,
                    allowDrawCellContent: true,
                    allowDrawCustomBorders: false
                });
            });

            page._tables.forEach((table) => {
                table.drawTo(doc, styles, {
                    allowDrawBorders: false,
                    allowDrawCellContent: false,
                    allowDrawCustomBorders: true
                });

Reply:
Okay, let's change the logic from "skip" to "allow".